skrf/vi/vna/hp8510c_sweep_plan.py

```diff
@@ -72,4 +72,4 @@ def matches_f_list(self, hz):
         True if the frequencies sweepned by self equal those in the list hz.
         """
         plan_hz = np.array(sorted(self.get_hz()))
-        return good
+        return True
```

Comment:
Love it :)

Reply:
haha, yeah, the binary search ax chopped a bit too hard that time